src/semantize/typing.nit

```diff
@@ -600,6 +600,25 @@ private class TypeVisitor
 		flow.set_var(self, variable, mtype)
 	end
 
+	# Find a most common type in `col`.
```

Comment:
What do you mean by “most common”? Also, the superlative form suggests that some types in `col` may be ignored in case of “conflict” (which is not the case). May I suggest “Find a common ancestor in `col`.”?